Update plugin documentation migration steps in Plugin Wiki Pages page (#4846)
jenkins-infra/jenkins.io: content/doc/developer/publishing/wiki-page.adoc

  title: Jenkins Wiki Exporter
---

WARNING: The Jenkins wiki was made 'read-only' in link:https://groups.google.com/d/msg/jenkinsci-dev/lNmas8aBRrI/eL3u7A6qBwAJ[October 2019]. 
In September 2021, link:https://www.jenkins.io/blog/2021/09/04/wiki-attacked/[Jenkins wiki was attacked] and as a result, the impacted server was permanently disabled and the site was rendered static.
Plugin documentation is now maintained in the GitHub repository of the plugin.
See the link:../documentation[plugin documentation guidelines] for details.

Plugin documentation is available from the link:https://plugins.jenkins.io[Jenkins plugins site] so that potential users can learn about the plugin without installing it.

== Migrating from Wiki to GitHub

> All plugin documentation have been converted to markdown and are stored in the link:https://github.com/jenkins-infra/plugins-wiki-docs/[plugins-wiki-docs] repository.

Reviewer: Singular rather than plural

```suggestion
All plugin documentation has been converted to markdown and is stored in the link:https://github.com/jenkins-infra/plugins-wiki-docs/[plugins-wiki-docs] repository.
```
